test/wasm2js/br_table_to_loop.2asm.js.opt

```diff
@@ -15,7 +15,7 @@ function asmFunc(importObject) {
  var nan = NaN;
  var infinity = Infinity;
  function $0() {
-  while (1) continue;
+  abort();
```

Comment:
Is this what we want to happen here?

Reply:
Yes, it's ok to do this since the only effect of an infinite loop is "trap" now, and we can replace traps. This is a smaller replacement so it does it.

Practically this is also fine since it replaces hanging forever with an immediate error, so it's not breaking useful code, it's just eagerly erroring basically.